src/Invoker.php

```diff
@@ -28,31 +29,45 @@ class Invoker
 {
     private $function;
     private $errorLogFunc;
+    private static $registeredFunction = [];
 
     /**
-     * @param callable $target      The callable to be invoked
-     * @param string $signatureType The signature type of the target callable,
-     *                              either "event" or "http".
+     * @param array|string|FunctionWrapper $target The callable to be invoked.
```

Comment:
My mistake - the function does not actually accept `FunctionWrapper`, only a string or array.

```suggestion
     * @param array|string $target The callable to be invoked.
```